docs/pages/enroll-resources/server-access/guides/recording-proxy-mode.mdx

```diff
@@ -3,9 +3,10 @@ title: Teleport Recording Proxy Mode
 description: Use Recording Proxy Mode to capture OpenSSH server activity
 ---
 
-Teleport Recording Proxy Mode was added to allow Teleport users
-to enable session recording for servers running `sshd`, which is helpful
-when gradually transitioning large server fleets to Teleport.
+Teleport Recording Proxy Mode was originally added to allow Teleport users
+to enable session recording for servers running `sshd`. Teleport supports
+recording `sshd` sessions in Recording Node Mode too so setting to Recording Proxy
+Mode is typically not required.
```

Comment:
How can we record openssh sessions at the node?

Reply:
In that mode it still records and yes that's technically at the proxy. I'll see about the language we had from the changelog.